jenkins/combirepo_run.jenkinsfile

```diff
@@ -316,7 +316,7 @@ pipeline {
           -o ${micDir} -l ${logFile} --tmp-dir ${cacheDir} -j ${params.JOBS_NUMBER} \
           \${greedy} \${mirror} \${rpm_patching} \${debug} \${verbose} \
           --regenerate-repodata --preferring-strategy big --skip-version-mismatch --mic-config ${micFile} \
-          -S libasan,asan-runtime-env,make,cpio,rpm-build --disable-libasan-preloading \
+          -S libasan,asan-runtime-env --mic-options="--pack-to=snapshot.tar" \
```

Comment:
`libasan` is required by `asan-runtime-env`, I think the latter package should be enough.

Reply:
Ok, I'll check it.

Reply:
Only asan-runtime-env is OK.